### DISCUSSION TITLE
Feature Request: FOR THE Business plan Edition Adressbook sharing with other users and group remote ids.

### DISCUSSION THREAD

**Discussion by lotze1234** [rustdesk/rustdesk#5462] [Feature Request]
would greatly appreciate the addition of a feature in the Address Book where addresses can be grouped together. Furthermore, it would be highly beneficial if these individual groups could be shared with team users. This functionality is of utmost importance. Regards

**Comment by xlionjuan** [comment]
Maybe OneNote, Keep or Word Online can help you

**Comment by yaroz** [comment]
This is the one feature that is keeping us from ditching anydesk and going 100% with RustDesk.  It's not bad when my team is managing 10 servers... but 200+ user desktops as well?

**Reply by lotze1234** [reply]
yes absolutely  ....

**Comment by dinger1986** [comment]
Can you please add to here https://github.com/rustdesk/rustdesk-server-pro/issues/49

**Comment by SergeyMy** [comment]
so it seems to be in the address book:
1. groups by tags.
2. assigning users membership in various groups.
3. inclusion of the visibility of strictly those participants who participate in the group.

as a bonus, customizable colored lights for each group;)

![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/2569776f-1f27-4e58-9198-eb6f67dc6cec)
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/d1b49d43-3bdf-4d47-83a2-77a8cee4c286)

**Reply by dinger1986** [reply]
And allow addressbooks to be shared by group

**Reply by SergeyMy** [reply]
it already depends entirely on the server API! so-called PRO features (as well as permission to change ID)

**Reply by dinger1986** [reply]
Yes of course. Was just confirming the asks for this

**Reply by lotze1234** [reply]
No, that's not what we need or want.

Sharing is not possible with team members (users on the server). Customizable colored lights for each group are not compatible with Mac and Linux versions.

We require a professional address list, similar to those found in TeamViewer, AnyDesk, pcvisit, etc. This is a crucial feature and an absolute necessity for genuine business use."

**Comment by SergeyMy** [comment]
> Customizable colored lights for each group are **not compatible** with Mac and **Linux** versions.

:rage: strange what am I doing wrong? :clap:
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/7d3c92a3-8dbc-4d15-a028-e362b06839a5)
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/5258783a-b312-4ac7-8e9e-96abc2b5507c)

**Reply by lotze1234** [reply]
Hmmm strange ... i have the newest version 1.2.2 and i cant find customizable colored lights for each groups. But anyway, it doesn't help us,  if we can't share the list with my teammembers....  thx

**Reply by dinger1986** [reply]
Like I said before can comments be added to the issue created on rustdesk-server-pro GitHub please? I linked to it above

**Comment by SergeyMy** [comment]
. I can afford to watch the project
. I can change it
. and I'm ready to quickly check

![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/51c2993a-e037-4a8f-9c4f-33cffd309047)
in 4+ minutes

PS check out today's nightly build
https://github.com/rustdesk/rustdesk/releases/tag/nightly
PSPS released release 1.2.3

**Reply by lotze1234** [reply]
baaah ohh shit thx 4 that. For MACOS 1 hour ago ... will test it :-)

**Comment by SergeyMy** [comment]
@lotze1234 
> share the list with my teammembers...

I repeat! this is a PRO service, everything is possible! or do it yourself, or order from the developer.
what dear @dinger1986  reminded you opaquely

**Reply by lotze1234** [reply]
we still have the pro version! Sharing is not possible!

**Reply by SergeyMy** [reply]
I try to call the PRO version like a PRO service
my opinion is that it is more true.

PS
question for thought:
RHEL 7 is a service or software?
CENTOS 7 - what is it?
there are no binary differences! :)

**Comment by SergeyMy** [comment]
> baaah ohh shit thx 4 that. For MACOS **1 hour ago** ... will test it :-)

release more than 10 hours ago, well, active users from Russia and Germany helped with localization (unlike other passive ones), which is probably why the developers rebuilt the assembly
